### PR TITLE
[DO-2370, 2371] Make changes to the look of the page of the sanctions register against company, country

### PR DESCRIPTION
**Pull request by Andrii17s**
Внести змінм в загальний вигляд сторінки відповідно до зразку
https://www.figma.com/file/ZsbdiuhEmsykHPNILBu3Qp/Sanctions?node-id=525%3A2

### CODE REVIEW

**Review by AlenaYanish — APPROVED** [review]

File: src/locales/uk/translation.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
   "company-sanction": "Реєстр санкцій щодо компаній",
   "companyID": "ID компанії",
   "companyInfo": "Iнформація про компанію",
-  "companyName": "Назва компанії",
+  "companyName": "Назва",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:54]: nope

File: src/locales/uk/translation.json [hunk 2]
```diff
@@ -129,7 +129,7 @@
...
-      "q15": "Для наповнення Реєстру  використано інформацію, що отримана в рамках реалізації ГО «Центр протидії корупції» свого права на доступ до публічної інформації, а також інформацію з відкритих загальнодоступних джерел, яка вже була оприлюднена. Вказана інформація не є такою, що має обмежений доступ та не містить персональних даних в розумінні Закону України «Про захист персональних даних» відповідно до частини 2 статті 5 вказаного Закону. Разом з тим, ми не можемо публікувати інформацію, яка відноситься до персональної, зокрема ідентифікаційні коди фізичних осіб.",
+      "q15": "Для наповнення Реєстру  використано інформацію, що отримана в рамках реалізації ГО «Центр протидії корупції» свого права на доступ до публічної інформації, а також інформацію з відкритих загальнодоступних джерел, яка вже була оприлюднена. Вказана інформація не є такою, що має обмежений доступ та не містить персональних даних в розумінні Закону України «Про захист персональних даних» відповідно до частини 2 статті 5 вказаного Закону. Разом з тим, ми не можемо публікувати інформацію, яка відноситься до персональної, зокрема ідентифікаційні коди фізичних осіб.",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:132]: extra space

File: src/locales/uk/translation.json [hunk 3]
```diff
@@ -371,6 +372,7 @@
   "systemLanguage": "Мова системи",
   "tariff": "тариф",
   "taxpayerNumber": "Номер платника податків",
+  "taxpayerNumberCompany": "Номер платника податків",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:375]: what different? I think we can merge his at one

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
pls... use i18n editor
https://github.com/jcbvm/i18n-editor

File: src/locales/en/translation.json [hunk 4]
```diff
@@ -52,6 +52,7 @@
   "companyID": "Company ID",
   "companyInfo": "Company's information",
   "companyName": "Company name",
+  "companyNameInTable": "Company name",
```
Comment (roman-tiukh) [inline src/locales/en/translation.json:55]: what difference?

File: src/locales/uk/translation.json [hunk 5]
```diff
@@ -50,8 +50,9 @@
   "company": "Компанія",
   "company-sanction": "Реєстр санкцій щодо компаній",
   "companyID": "ID компанії",
-  "companyInfo": "Iнформація про компанію",
+  "companyInfo": "Інформація про компанію",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:53]: ?

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**